### PR TITLE
Improving kepcotrend code

### PR DESCRIPTION
**Pull request by mirca**

### CODE REVIEW

**Comment by mirca** [comment]
cc @barentsen @gully

**Comment by gully** [comment]
It looks good to me, without actually running it.

File: pyke/kepcotrend.py [hunk 1]
```diff
@@ -16,36 +16,31 @@
...
-    for i in range(len(bad_data)):
-        if bad_data[i] == True:
+    for i in range(len(good_data)):
+        if good_data[i]:
             newflux[i] = flux[j]
             j += 1
-        elif bad_data[i] == False:
+        else:
             newflux[i] = np.nan
     return newflux
```
Comment (barentsen) [inline pyke/kepcotrend.py]: I think you can simplify this entire function as follows:
```
newflux = np.empty(len(good_data))
newflux[:] = np.nan
newflux[good_data] = flux
```

Comment (mirca) [inline pyke/kepcotrend.py]: absolutely!

**Pull request merged**